File: zadanie1/zadanie1.py
def main():
	a = int(input("Podaj pierwszą liczbę: "))
	b = int(input("Podaj drugą liczbę: "))
	while b > 0:
		a, b = b, a%b
	return a


if __name__ == "__main__":
	print(main())

File: zadanie1/zadanie1.c
#include <stdio.h>

int main(){
  int a,b,c;
  printf("Podaj pierwszą liczbę: ");
  scanf("%d", &a);
  printf("Podaj drugą liczbę: ");
  scanf("%d", &b);
  while (b > 0) {
		c = a%b;
		a = b;
		b = c;
  }
  printf("%d", a);

}
